layouts/joomla/content/category_default.php

```diff
@@ -74,7 +75,14 @@
 			<div class="category-desc">
 				<?php if ($params->get('show_description_image') && $category->getParams()->get('image')) : ?>
 					<?php $alt = empty($category->getParams()->get('image_alt')) && empty($category->getParams()->get('image_alt_empty')) ? '' : 'alt="' . htmlspecialchars($category->getParams()->get('image_alt'), ENT_COMPAT, 'UTF-8') . '"'; ?>
-					<img src="<?php echo $category->getParams()->get('image'); ?>" <?php echo $alt; ?>>
+					<img src="<?php echo $category->getParams()->get('image'); ?>" <?php echo $alt; ?>
+						<?php
+							$img = HTMLHelper::cleanImageURL($category->getParams()->get('image'));
```

Comment:
It is better to move the initializations to the top, where $category is written. The HTML part is chaotic and hard to read (not your fault, just typical PHP :)